expertsystem/data.py

```diff
@@ -149,6 +149,30 @@ class EdgeQuantumNumbers:  # pylint: disable=too-many-instance-attributes
         edge_qn_type.__module__ = "expertsystem.data"
 
 
+# for static typing
+EdgeQuantumNumber = Union[
```

Comment:
We may want to split up this module into something like `types` (which is used internally) and `particles` (which is the 'starting point' interface of the `expertsystem`). See also #196 and also the new `amplitude.model` module (#287).

Reply:
Yes, we can address that in #196